pkg/auth/handler/forgotpwd/reset.go

```diff
@@ -33,6 +33,9 @@ func AttachForgotPasswordResetHandler(
 	server.Handle("/forgot_password/reset_password", &ForgotPasswordResetHandlerFactory{
 		authDependency,
 	}).Methods("OPTIONS", "POST")
+	server.Handle("/forgot_password/reset_password_form", &ForgotPasswordResetFormHandlerFactory{
+		authDependency,
+	}).Methods("POST", "GET")
```

Comment:
Do we need support "POST"?

Reply:
Yes, https://github.com/SkygearIO/forgot_password/blob/master/forgot_password/handlers/reset_password.py#L238